index.bs

```diff
@@ -99,19 +105,19 @@ class AV1SampleEntry extends VisualSampleEntry('av01') {
 
-Optional boxes not specifically mentioned here can be present, in particular those indicated in the definition of the [=VisualSampleEntry=] in [[ISOBMFF]].
+As specified in [[!ISOBMFF]], the width and height in the [=VisualSampleEntry=] are specified in square pixels. If the video pixels are not square, then a 'pasp' box MUST be included and the track header width and height SHOULD match the values of [=max_frame_width_minus_1=] + 1 and [=max_frame_height_minus_1=] + 1 after the 'pasp' ratio has been applied.
 
-<dfn>width</dfn> and <dfn>height</dfn> of the [=VisualSampleEntry=] SHALL be encoded as an upper bound of the frame size of the video samples associated with that entry. AV1 supports on-the-fly decoding of dynamic video resolutions, so these fields SHALL contain the highest values of the portion of the video stream associated with that entry. The width and height in the [=VisualSampleEntry=] are specified in square pixels. If the video pixels are not square, then a 'pasp' box MUST be included. The 'pasp' box and the track header width and height SHOULD match the value of the render size given in the associated samples.
+The <dfn>compressorname</dfn> field of the [=VisualSampleEntry=] is an informative name. It is formatted in a fixed 32-byte field, with the first byte set to the number of bytes to be displayed, followed by that number of bytes of displayable data, followed by padding to complete 32 bytes total (including the size byte). The size field MAY be set to 0. The value "\012AOM Coding" is RECOMMENDED; the first byte is a count of the remaining bytes, here represented by \012, which (being octal 12) is decimal 10, the number of bytes in the rest of the string.
 
-ISSUE: Given that the render size could change within a stream, it is hard to provide a one-to-one mapping, thus the "should".
+NOTE: Parsers may ignore the value of the compressorname field. It is specified in this document simply for legacy and backwards compatibility reasons.
 
-<dfn>compressorname</dfn> of the [=VisualSampleEntry=] is an informative name. It is formatted in a fixed 32-byte field, with the first byte set to the number of bytes to be displayed, followed by that number of bytes of displayable data, followed by padding to complete 32 bytes total (including the size byte). The size field MAY be set to 0. The value "\012AOM Coding" is RECOMMENDED; the first byte is a count of the remaining bytes, here represented by \012, which (being octal 12) is decimal 10, the number of bytes in the rest of the string.
+The <dfn>config</dfn> field SHALL contain an [=AV1CodecConfigurationBox=] that applies to the samples associated with this sample entry.
 
-NOTE: Parsers may ignore the value of the compressorname field. It is specified in this document simply for legacy and backwards compatibility reasons.
+NOTE: Multiple [=AV1SampleEntry=] may be required when the track contains samples having requiring different AV1ConfigurationBox.
```

Comment:
Maybe reword this:

Multiple instances of [=AV1SampleEntry=] may be required when the track contains samples requiring a AV1ConfigurationBox with different characteristics.